hw/dv/tools/testplans/shadow_reg_errors_testplan.hjson

```diff
@@ -0,0 +1,30 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  entries: [
+    {
+      name: shadow_reg_errors
+      desc: '''
+            Verify shadow registers' update and storage errors.
+            - Randomly choose to issue reset. A reset will clear all the internal stored values and
+              phases trackers in shadow registers.
+            - Select all the shadow registers and randomly select the rest of the non-excluded
```

Comment:
nit: I think we can be clearer on the working here also:
"randomly select the rest of the non-excluded registers" if you select the rest of them what is then random?

suggestion:

> 
> The shadow registers and the non excluded registers are shuffled and a register is selected at random. 
> Randomized values is then written to the selected register

Reply:
Thank you for your detailed suggestion I see this is confusing.. Yeah but here I am not selecting one register to write. I am trying to select all the shadow registers, plus a random amount of the rest of the registers, then write them all with random values.
Are you okay with this?
>- Select all the shadow registers and a random amount of the rest of the non-excluded registers. Shuffle and write random values are written to these selected registers.

Reply:
That is ok with me 👍